app/store/models/User.ts

```diff
@@ -82,6 +82,7 @@ interface User {
   linkedinId?: string;
   actionGrant?: ActionGrant;
   achievements: Achievement[];
+  achievementScore: number;
```

Comment:
```suggestion
  achievementsScore: number;
```
Please, let's be consistent here

Also; not really the right place to discuss this but achievements != trophies